Automattic/wp-calypso
# Checkout: add additional Jetpack URL slugs to system decider

In #44039, Jetpack support was added to the new Checkout flow for testing. Our logs identified a number of unsupported URL slugs from Jetpack.com and `/wp-admin/` links (not Calypso `/plans/`) that were being caught by the system decider. This adds those additional slugs.

**To test:**
- create a jetpack site at jurassic.ninja
- visit `calypso.localhost:3000/checkout/{your jetpack site slug}/jetpack_search_monthly`
- make sure that the monthly search product is correctly added to the cart and that composite checkout is displayed
- empty your cart and repeat, replacing `jetpack_search_monthly` in the url with each of the following: `jetpack_anti_spam`, `jetpack_antispam_monthly`, `jetpack_backup_daily_monthly`, `jetpack_backup_realtime_monthly`, `jetpack_search_monthly`, `jetpack_premium`
